v3io/dataplane/request.py

```diff
@@ -104,7 +104,8 @@ def encode_get_object(container_name, access_key, kwargs):
     offset = kwargs.get("offset")
 
     # if the append flag is passed, add a range header
-    if offset:
+    # offset=0 also requires a range because we may only have the size.
+    if offset is not None:
```

Comment:
Need to add a regression test.

Reply:
Should probably be
```suggestion
    num_bytes = kwargs.get("num_bytes")
    if offset or num_bytes:
        offset = offset or 0
```

Reply:
Because offset is said to default to 0, so it's not correct for there to be a difference between passing `offset=0` and not passing it at all.